packages/cli/openapi-parser/src/v3/converters/convertSchemas.ts

```diff
@@ -148,6 +148,13 @@ export function convertSchemaObject(
 
     // primitive types
     if (schema === "boolean" || schema.type === "boolean") {
+        const literalValue = getExtension<boolean>(schema, FernOpenAPIExtension.BOOLEAN_LITERAL);
+        if (literalValue !== undefined) {
+            return SchemaWithExample.literal({
+                value: LiteralSchemaValue.boolean(literalValue),
+                description,
+            });
```

Comment:
we're going to want to call something like `wrapPrimitive`

Reply:
`wrapPrimitive` handles wrapping in nulllable/optional depending on what has been specified